Спринт 14

cyberschoolinmoscow opened the pull request:
1) Реализована логика воспроизведения отрывка трека на экране «Аудиоплеер».
2) Реализована логика выполнения запроса на экране «Поиск» при помощи debounce.
3) Добавлен ProgressBar для индикации процесса выполнения поискового запроса на экране «Поиск».
4) Добавлен debounce для обработки нажатия на элемент списка результатов поиска.

cyberschoolinmoscow committed:
1) Реализована логика воспроизведения отрывка трека на экране «Аудиоплеер».
2) Реализована логика выполнения запроса на экране «Поиск» при помощи debounce.
3) Добавлен ProgressBar для индикации процесса выполнения поискового запроса на экране «Поиск».
4) Добавлен debounce для обработки нажатия на элемент списка результатов поиска.

cyberschoolinmoscow committed:
1) Реализована логика отражения прогресса воспроизведения отрывка трека на экране «Аудиоплеер».

cyberschoolinmoscow committed:
1) Отредактированы кнопки воспроизведения, паузы и отметки понравилось на экране «Аудиоплеер» в соответствии с макетом.

DmitrijEgorow commented:
- [ ] отсутствует progress bar

https://github.com/user-attachments/assets/f90a0788-9aa2-4efa-9e5c-96d834d26865

DmitrijEgorow commented:
- [ ] Зачем внизу чёрная полоска не в цвет фона?
- [ ] Почему трек автоматически показан будто в избранном?
 
<img width="288" alt="Screenshot 2024-07-18 at 23 53 13" src="https://github.com/user-attachments/assets/6f5e8c02-6ed9-4f56-aac8-9fd271c7fde3">

DmitrijEgorow commented:
- [ ] Неконсистентное состояние, пропала кнопка для очистки истории

https://github.com/user-attachments/assets/052921fa-0c37-4e86-8fa3-a0077a908748

DmitrijEgorow commented:
- [ ] Автор может быть длинным, нужно обрабатывать такой случай
<img width="237" alt="Screenshot 2024-07-18 at 23 57 28" src="https://github.com/user-attachments/assets/8c2d6ef6-9c00-4604-b791-3f66c0d42a4f">

DmitrijEgorow commented:
- [ ] Цвет иконок светлее, чем в фигме
![Screenshot_20240719_000240](https://github.com/user-attachments/assets/698507b1-ac0b-4e9f-a749-7e8ca0a756d6)

DmitrijEgorow commented:
- [ ] Неправильное изображение плейсхолдера
![Screenshot_20240719_000444](https://github.com/user-attachments/assets/1fa6cd5a-aa32-41ee-b6b7-1dacaaa01e2f)

cyberschoolinmoscow committed:
Исправлено:
1) Отображение плейсхолдера
2) Цвет иконок в экране Настройки
3) Обработка длинной строки названия альбома на экране Плеер
4) Замечания по верстке экрана Плеер
5) Замечания по логике работы поиска на экране Поиск
